ports/boost-math/vcpkg.json

```
@@ -55,5 +56,10 @@
       "name": "boost-throw-exception",
       "version>=": "1.85.0"
     }
-  ]
+  ],
+  "features": {
+    "c99-tr1": {
+      "description": "Build with C99 and TR1 Functions"
+    }
```

```suggestion
    "legacy": {
      "description": "Build the legacy C99 and TR1 libraries"
    }
```
I would call the feature this way. But I would also say to wait for MS review.